exercises/go-counting/src/test/java/GoCountingTest.java

```diff
@@ -12,167 +12,167 @@
 import org.junit.rules.ExpectedException;
 
 public class GoCountingTest {
-	
+
 	@Rule
-	public ExpectedException expectedException = ExpectedException.none(); 
-	
+	public ExpectedException expectedException = ExpectedException.none();
+
 	String board5x5 = "  B  \n" +
 	                  " B B \n" +
 	                  "B W B\n" +
 	                  " W W \n" +
 	                  "  W  ";
-	
+
 	@Test
 	public void blackCorner5x5BoardTest() {
 		GoCounting gocounting = new GoCounting(board5x5);
-		
+
 		Set<Point> territory = new HashSet<>();
 		territory.add(new Point(0, 0));
 		territory.add(new Point(0, 1));
 		territory.add(new Point(1, 0));
-		
+
 		assertEquals(Player.BLACK, gocounting.getTerritoryOwner(0, 1));
 		assertEquals(territory, gocounting.getTerritory(0, 1));
 	}
-	
+
 	@Ignore("Remove to run test")
 	@Test
 	public void whiteCenter5x5BoardTest() {
 		GoCounting gocounting = new GoCounting(board5x5);
-		
+
 		Set<Point> territory = new HashSet<>();
 		territory.add(new Point(2, 3));
-		
-		assertEquals(Player.WHITE, gocounting.getTerritoryOwner(2, 3));
+
+		assertEquals(Player.NONE, gocounting.getTerritoryOwner(2, 3));
 		assertEquals(territory, gocounting.getTerritory(2, 3));
 	}
-	
+
 	@Ignore("Remove to run test")
 	@Test
 	public void openCorner5x5BoardTest() {
 		GoCounting gocounting = new GoCounting(board5x5);
-		
+
 		Set<Point> territory = new HashSet<>();
 		territory.add(new Point(0, 3));
 		territory.add(new Point(0, 4));
 		territory.add(new Point(1, 4));
-		
-		assertEquals(Player.NONE, gocounting.getTerritoryOwner(1, 4));
+
+		assertEquals(Player.BLACK, gocounting.getTerritoryOwner(1, 4));
```

Comment:
It looks like this Player should be `NONE` according to the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/go-counting/canonical-data.json) so I don't think this test needs to change at the minute :slightly_smiling_face: